en/asset/model/image-location-resolution.md

```diff
@@ -0,0 +1,35 @@
+# Cocos Creator 图像位置解析算法
```

Comment:
Does this need to be English?

Reply:
Yes, when this pr is merged, I will add it to the list of pending translations for the issue #5638.